extensions/amp-accordion/1.0/animations.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable local/no-style-property-setting */
+
+const MAX_TRANSITION_DURATION = 500; // ms
+const MIN_TRANSITION_DURATION = 200; // ms
+const EXPAND_CURVE = 'cubic-bezier(0.47, 0, 0.745, 0.715)';
+const COLLAPSE_CURVE = 'cubic-bezier(0.39, 0.575, 0.565, 1)';
+
+/**
+ * @param {!Element} content
+ * @return {!UnlistenDef}
+ */
+export function animateExpand(content) {
+  return animate(content, () => {
+    const {
+      height: oldHeight,
+      opacity: oldOpacity,
+      overflowY: oldOverflowY,
+    } = content.style;
+
+    // Measure the expanded height. This is relatively heavy with a sync
+    // layout. But no way around it. The hope that the `commitStyles` API
+    // may eventually make this unneeded.
+    content.style.height = '0';
```

Comment:
Optional: `setStyles` from `style.js` can be used here.

Reply:
Oh. We can? I though Preact components were complaining about those utils. I'll try.

Reply:
They are used in `FitText`

Reply:
Thanks! That worked. I removed the config changes as well.